applications/dashboard/models/class.logmodel.php

```diff
@@ -430,19 +435,25 @@ public function getWhere($where = false, $orderFields = '', $orderDirection = 'a
      * Get the count of log entries matching a query.
      *
      * @param array $where The filter.
+     * @param bool|array $orWhere
      * @return int Returns the count.
      */
-    public function getCountWhere($where) {
+    public function getCountWhere($where, $orWhere = false) {
```

Comment:
The changes we're making here are very similar to what we're making in `getWhere`, so I have the same concerns (`null` versus `false` and is-array versus is-truthy).